ui/components/ui/text-field/index.scss

```diff
@@ -2,3 +2,9 @@
 .MuiInputBase-input {
   color: var(--color-text-default) !important;
 }
+
+.MuiFormHelperText-root.Mui-error {
+  @include H7;
+
+  color: var(--color-error-default) !important;
+}
```

Comment:
Fixes non design system approved color for input error text

<img width="1339" alt="Screen Shot 2022-06-14 at 4 39 20 PM" src="https://user-images.githubusercontent.com/8112138/173706839-15b5c86d-b634-4af6-9adf-d543fae0fedd.png">